docs/configuration.md

```diff
@@ -7,6 +7,93 @@ nav_order: 10
 Modular has minimal configuration because of its philosophy. However there is a
```

Comment:
We should update the wording here now IMO. It's a bit confusing saying "there is a set of minimum configuration required" and then going on to talk about some optional configuration.

Maybe something like:

```markdown
Modular has minimal configuration because of its philosophy, but some behaviours can be controlled.
```